src/index.ts

```diff
@@ -0,0 +1,211 @@
+  public static instance: EppoJSClient = new EppoJSClient(
+    new MemoryOnlyConfigurationStore(),
+    undefined,
+    true,
+  );
+  public static initialized = false;
+
+  public getStringAssignment(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: Record<string, string | number | boolean>,
+    defaultValue: string,
+  ): string {
+    EppoJSClient.getAssignmentInitializationCheck();
+    return super.getStringAssignment(flagKey, subjectKey, subjectAttributes, defaultValue);
+  }
+
+  public getBoolAssignment(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: Record<string, string | number | boolean>,
+    defaultValue: boolean,
+  ): boolean {
+    EppoJSClient.getAssignmentInitializationCheck();
+    return super.getBoolAssignment(flagKey, subjectKey, subjectAttributes, defaultValue);
+  }
+
+  public getIntegerAssignment(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: Record<string, string | number | boolean>,
+    defaultValue: number,
+  ): number {
+    EppoJSClient.getAssignmentInitializationCheck();
+    return super.getIntegerAssignment(flagKey, subjectKey, subjectAttributes, defaultValue);
+  }
+
+  public getNumericAssignment(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: Record<string, string | number | boolean>,
+    defaultValue: number,
+  ): number {
+    EppoJSClient.getAssignmentInitializationCheck();
+    return super.getNumericAssignment(flagKey, subjectKey, subjectAttributes, defaultValue);
+  }
+
+  public getJSONAssignment(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: Record<string, string | number | boolean>,
+    defaultValue: object,
+  ): object {
+    EppoJSClient.getAssignmentInitializationCheck();
+    return super.getJSONAssignment(flagKey, subjectKey, subjectAttributes, defaultValue);
+  }
+
+  private static getAssignmentInitializationCheck() {
+    if (!EppoJSClient.initialized) {
+      console.warn('Eppo SDK assignment requested before init() completed');
+    }
+  }
+}
+
+/**
+ * Initializes the Eppo client with configuration parameters.
+ * This method should be called once on application startup.
+ * @param config - client configuration
+ * @public
+ */
+export async function init(config: IClientConfig): Promise<IEppoClient> {
+  validation.validateNotBlank(config.apiKey, 'API key required');
+  try {
+    // If any existing instances; ensure they are not polling
+    if (EppoJSClient.instance) {
+      EppoJSClient.instance.stopPolling();
+    }
+
+    const configurationStore = new HybridConfigurationStore(
+      new MemoryStore<Flag>(),
+      new VercelAsyncStore<Flag>(
+        config.vercelParams.edgeConfig,
+        config.vercelParams.edgeConfigStoreId,
+        config.vercelParams.vercelApiToken,
+      ),
+    );
+
+    await configurationStore.init();
+    EppoJSClient.instance.setConfigurationStore(configurationStore);
+    EppoJSClient.instance.disableAssignmentCache();
```

Comment:
Why not use an [in-memory](https://github.com/Eppo-exp/js-client-sdk-common/blob/main/src/client/eppo-client.ts#L439) one to at least not log duplicate assignments for the single request?

Would it make sense to use another edge config key as part of a [custom](https://github.com/Eppo-exp/js-client-sdk-common/blob/main/src/client/eppo-client.ts#L447C3-L447C13) assignment cache?